# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/WindowSurface.h
# Commit: 2b20b560fe61
# Full Hash: 2b20b560fe614c26fe779367c3323addaa55122a
# Author: stransky <stransky@redhat.com>
# Date: 2021-06-20 09:32:48
# Description:
#   Bug 1716796 [Wayland] Poll mozcontainer remap state, r=rmader
#   
#   - Poll mozcontainer remap state by moz_container_wayland_remapped()
#   - Remove WindowSurface::Reset()
#   - Remove moz_container_wayland_set_window_surface() as it adds extra ref at WindowSurface() and causes cyclic dependency.
# ==============================================================================

diff -r 668013f3e201 -r 2b20b560fe61 widget/gtk/WindowSurface.h
--- a/widget/gtk/WindowSurface.h	Sat Jun 19 07:06:41 2021 +0300
+++ b/widget/gtk/WindowSurface.h	Sat Jun 19 09:33:19 2021 +0000
@@ -31,10 +31,6 @@
   // Whether the window surface represents a fallback method.
   virtual bool IsFallback() const { return false; }
 
-  // Reset internal states (stored surfaces etc.) as attached
-  // Gtk widget was hidden/shown. Used on Wayland only.
-  virtual void Reset() {}
-
  protected:
   virtual ~WindowSurface() = default;
 };